homeassistant/components/transmission/config_flow.py

```diff
@@ -84,7 +78,8 @@ async def async_step_user(
 
             if not errors:
                 return self.async_create_entry(
-                    title=user_input[CONF_NAME], data=user_input
+                    title=f"{user_input[CONF_HOST]}:{user_input[CONF_PORT]}",
```

Comment:
One thing I am now a bit doubting about, this is also going to be the device name. So the entities will be called `sensor.192_168_0_123_1234_torrents`. What do you think about maybe defaulting the name to `Transmission` and have people rename the entries as they see fit? (Like, when you only have 1 Transmission instance (as probably 90% of all users) having the IP in the entity ids looks kinda strange. So that way we are kinda forcing the user to rename it if they want normal entity_ids. When you have more Transmission setups you can always rename the entry and the devices all at once to make sure you know which one is which).

But this is just my opinion, what do you think?

Reply:
I don't mind going with this approach. It's just that in other integrations like "glances" we are naming the entry like this "host:port". So, better to be consistent in all the integrations whether to use the 'host' and 'port' in the title and device name or do as you are suggesting here.

Reply:
Also asked Erik, and he also thinks having IP in the entities like that isn't ideal, I think changing it would be best